bin/update_clinvar_schema.py

```diff
@@ -0,0 +1,80 @@
+import gzip
+import argparse
+import sys
+import urllib.request
+import subprocess
+import shutil
+import os
+
+SCHEMA_LOCATION_ATTRIBUTE = 'SchemaLocation'
+
+def launch():
+	parser = ArgParser(sys.argv)
+	with gzip.open(parser.input_file, 'rt') as xml_file:
```

Comment:
This function could be split up a bit more. For example there could be a function to get the schema_url variable, another to download the file, and another to generate the jaxb classes.